docs/tutorial_aretomo3.md

```diff
@@ -0,0 +1,249 @@
+---
+tocdepth: 2
+---
+(tutorial-aretomo3)=
+# AreTomoLive Demo using data from the CryoET Data Portal
+
+This guide introduces the AreTomoLive [[1]](#references) pipeline to perform cryoET data preprocessing, spanning motion
+correction of the acquired tilt-movies to contrast enhancement of the reconstructed tomograms. The AreTomoLive pipeline
+consists of two GPU-accelerated packages, AreTomo3 and DenoisET. AreTomo3 provides a fully automated,
+multi-GPU-accelerated workflow to reconstruct tomograms from raw tilt-series with quality metrics reported. DenoisET
+implements the machine learning algorithm Noise2Noise and is designed to run in parallel with AreTomo3 to perform
+contrast enhancement. This demo provides instructions for running both packages on a dataset of purified synaptosomes,
+which is available on the CryoET Data Portal.
+
+## Background Information
+
+### Dataset
+
+The dataset of purified synaptosomes used in this demo is [available on the CryoET Data Portal](https://cryoetdataportal.czscience.com/datasets/10447), with some additional details below:
+
+- The sample of purified synaptosomes was generated from rat hippocampi.
+- The full dataset includes 76 runs. Each run contains the raw movies, the motion-corrected tilt-series, two types of
+  tomograms, and membrane segmentations. Only the raw movies are needed to test AreTomoLive.
+- The tomograms on the CryoET Data Portal were also reconstructed by AreTomo3 and denoised by DenoisET so can be
+  compared with the tomograms from this demo.
+
+The options to download this dataset are:
+
+1. [AWS CLI (used in this demo)](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html)
+2. [The CryoET Data Portal API](https://chanzuckerberg.github.io/cryoet-data-portal/python-api.html)
+
+### Software
+
+Both software packages used in this tutorial are publicly available on GitHub:
+
+1. AreTomo3: [https://github.com/czimaginginstitute/AreTomo3](https://github.com/czimaginginstitute/AreTomo3)
+
+   The latest version on GitHub and in this demo is 2.1.3. You can download a [pre-compiled executable from GitHub](https://github.com/czimaginginstitute/AreTomo3/releases/tag/v2.1.3).
```

Comment:
The executable is version 2.1.3, and the latest version on GitHub is also 2.1.3 so I changed the text here to read 2.1.3 instead of 2.1.0.